application/clicommands/RescanCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/* Icinga Web 2 X.509 Module | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\X509\Clicommands;
+
+use Icinga\Application\Logger;
+use Icinga\Module\X509\CertificateUtils;
+
+class RescanCommand extends ScanCommand
+{
+    /**
+     * Rescans already known IP and port to find X.509 certificates.
+     *
+     * USAGE
+     *
+     * icingacli x509 rescan [options]
+     *
+     * OPTIONS:
+     *
+     *   --job=<name>          Rescan known IPs which belong to the given job.
+     *   --last-scan='2days'   Rescan targets whose last scan is older than the specified time
+     */
+    public function indexAction()
+    {
+        $lastScan = $this->params->shift('last-scan');
+        if ($lastScan) {
+            if (! ($value = strtotime($lastScan, 0))) {
+                throw new \InvalidArgumentException(sprintf(
+                    "The specified last scan time is in an unknown format: '%s'",
+                    $lastScan
+                ));
+            }
+
+            $lastScan = $value;
+        }
+
+        $job = $this->getJob();
+        $job
+            ->setRescan(true)
+            ->setLastScan($lastScan);
+
+        $finishedTargets = $job->run();
```

Comment:
This is basically duplicate code except for `! $finishedTargets` vs. `$finishedTargets === null` and `Scanned` vs. `Rescanned`. I would use the same code for both and speak of `Scanned`.